src/NexusMods.DataModel/Games/AGame.cs

```diff
@@ -100,8 +101,7 @@ private List<GameInstallation> GetInstallations()
     /// <param name="locator">The locator used to find this game installation.</param>
     /// <param name="installation">An installation of the game found by the <paramref name="locator"/>.</param>
     /// <returns></returns>
-    protected abstract IEnumerable<KeyValuePair<GameFolderType, AbsolutePath>> GetLocations(
-        IFileSystem fileSystem,
+    protected abstract IReadOnlyDictionary<GameFolderType, AbsolutePath> GetLocations(IFileSystem fileSystem,
         IGameLocator locator,
```

Comment:
The `locator` field is currently unused. Do we have a known use case where we need to search for Game B from Game A?

Reply:
Yeah, not sure on that one, might be a leftover from previous versions of the code, but I can't be sure there aren't some valid use cases

Reply:
I think that's probably an accidental leftover, since we already have a `GameLocatorResult` available as a parameter.

@erri120 seems to have written that code; can you confirm if it's intended?

Reply:
The `IGameLocator` interface is pretty old, you can probably remove it if it's not used.